docker-compose.yml

```diff
@@ -121,7 +121,7 @@ services:
       - ./s3/pkg/datastore/yig/conf/:/etc/yig
       - /etc/ceph:/etc/ceph
       - ./examples/multi-cloud.conf:/etc/multi-cloud/multi-cloud.conf
-      - ./s3/pkg/conf/tidb.sql:/etc/config/tidb/s3.sql
+      - ./s3/pkg/conf/tidb.sql:/etc/config/tidb/tidb.sql
```

Comment:
will this require changes in initdb.sh? Or else, S3 functionalities will be impacted

Reply:
initdb.sh already has the change.

Reply:
ok